app/components/drugList/drugList.controller.js

```diff
@@ -6,79 +6,46 @@
  *
  * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
  * graphic logo is a trademark of OpenMRS Inc.
- */		
-DrugsListController.$inject = 
-	['loadDrugs', '$location', 'openmrsRest'];
+ */
 	
-export default function DrugsListController(loadDrugs, $location, openmrsRest){
+export default function DrugsListController(){
 	
 	var vm = this;
-	
+
+	//Properties for list component
+	vm.resource = "drug";
+	vm.redirectionParam = "drug";
+	vm.limit = 10; //Default
+	vm.columns= [
+		{
+			"property": "name",
+			"label": "Name"
+		}];
+	vm.actions = [
```

Comment:
Could we please introduce default values for label and icon for common actions such as edit, retire, unretire, purge, view? It would save us time by being able to just say vm.actions = [ { "action":"edit" }, { "action": "retire" } ];